libs/wingc/src/ast.rs

```diff
@@ -259,7 +259,7 @@ pub struct Class {
 	pub name: Symbol,
 	pub fields: Vec<ClassField>,
 	pub methods: Vec<(Symbol, FunctionDefinition)>,
-	pub constructor: Constructor,
+	pub constructor: Option<Constructor>,
```

Comment:
Is it possible we can create a default `Constructor` value when parsing (if the user doesn't provide an `init()` function) instead of making this optional? I'd expect the `Constructor` to have an empty list of parameters and an empty body. This will reduce the number of edge cases other parts of the compiler need to handle.